### PR TITLE
Add plotObsMon

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
These changes to obs-monitor include:

- new `plotObsMon` script
- `gfs_plot.yaml` driver script 
- updated (and renamed) `oznHoriz.yaml` plot template.

Much more needs to be done but I wanted to have folks review this approach before I got much further along.  The basic process flow for `plotObsMon` is:

1. Ingest plot instructions from `gfs_plot.yaml`
2. Use `wxflow` to modify requested plot template(s) and produce yaml plot file(s)
3. Use `eva` and the yaml plot files to generate plots

### CODE REVIEW

File: parm/gfs/oznHoriz.yaml
Comment (CoryMartin-NOAA) [inline parm/gfs/oznHoriz.yaml]: Guessing these are an oversight since it is in draft?

File: parm/gfs/oznHoriz.yaml
Comment (CoryMartin-NOAA) [inline parm/gfs/oznHoriz.yaml]: Note to fill in variables for titles/filenames here too

File: ush/logger.py
Comment (CoryMartin-NOAA) [inline ush/logger.py]: This is pulled directly from eva I'm assuming? I think Rahul has a logger that is similar in wxflow that we can probably leverage.

File: ush/plotObsMon
Comment (CoryMartin-NOAA) [inline ush/plotObsMon]: It might be easier to read/add extra things if doing something like this:
```
config = {
   'SAT': satname,
  'SENSOR': instrument,
...
}
```
Then you can still do:
`config['PDATE'] = cycle_tm` down below

File: ush/plotObsMon
Comment (CoryMartin-NOAA) [inline ush/plotObsMon]: suggest:
```
date_str = f"PDATEm{x*6}"
```

File: ush/plotObsMon
Comment (CoryMartin-NOAA) [inline ush/plotObsMon]: Probably need a more robust path here, and should use `os.path.join` rather than adding strings for paths.

File: ush/plotObsMon
Comment (CoryMartin-NOAA) [inline ush/plotObsMon]: since eva is python, we should be able to run it from python. Not 100% sure on that, but I also think wxflow has a execute method that should be used here instead at the very least

File: ush/plotObsMon
Comment (CoryMartin-NOAA) [inline ush/plotObsMon]: use `os.remove` on the file

**Review by CoryMartin-NOAA — COMMENTED** [review]
A few minor comments, but this is shaping up nicely in my opinion

File: parm/gfs/oznHoriz.yaml
Comment (EdwardSafford-NOAA) [reply parm/gfs/oznHoriz.yaml]: Indeed.  I proved the concept and figured I'd clean up later and forgot about it.   I'll correct that.

File: parm/gfs/oznHoriz.yaml
Comment (EdwardSafford-NOAA) [reply parm/gfs/oznHoriz.yaml]: Noted and done.

File: ush/plotObsMon
Comment (EdwardSafford-NOAA) [reply ush/plotObsMon]: Done.

File: ush/plotObsMon
Comment (EdwardSafford-NOAA) [reply ush/plotObsMon]: Done here and elsewhere in `plotObsMon`.

File: ush/plotObsMon
Comment (EdwardSafford-NOAA) [reply ush/plotObsMon]: Done.  I also moved the base path to the data to the` gfs_plots.yaml` file.  There are questions on locating/access the data that I'm intentionally leaving alone for now.  It's sufficiently complicated that I figure to tackle that as a separate issue and with consultation.

Comment (CoryMartin-NOAA) [reply ush/plotObsMon]: Yeah agreed, that should be separate. Alongside of actually moving around files in other scripts.

File: ush/plotObsMon
Comment (EdwardSafford-NOAA) [reply ush/plotObsMon]: I was able to get it to work in python by importing `eva_driver` and then calling `eva_driver.eva()`. 

My genYAML call here is actually a method in this file which makes two calls to wxflow routines.  I've since renamed it to genYaml to be consistent with the other methods in this file.

File: ush/plotObsMon
Comment (EdwardSafford-NOAA) [reply ush/plotObsMon]: Done.

**Review by CoryMartin-NOAA — APPROVED** [review]
Looks good, I'll wait for others to review/have a look before merging. Thanks @EdwardSafford-NOAA !

**Comment by EdwardSafford-NOAA** [comment]
@CoryMartin-NOAA Great!  I'm going to open another issue to update the other template files to work with this new scheme and get to work on that.

**Review by kevindougherty-noaa — APPROVED** [review]
Looks great. Cool to see you take advantage of those new tools

**Pull request merged**
